docs/pages/management/diagnostics/tracing.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Distributed Tracing
+description: How to enable tracing within Teleport.
+---
+
+Teleport leverages [OpenTelemetry](https://opentelemetry.io/) to generate traces
+and export them to any [OpenTelemetry Protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/otlp/)
+capable exporter. In the event that your telemetry backend doesn't have support receiving OTLP traces you may be able to
+leverage the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to proxy traces from OTLP
+to a format that your telemetry backend accepts.
+
+## Configure Teleport
+
+In order to enable tracing in Teleport, add the following section in `teleport.yaml`. For a detailed description of
```

Comment:
Is this the `teleport.yaml` for a specific `teleport` instance you want to enable tracing for, or do you only need to enable tracing on one `teleport` instance, e.g., the Auth or Proxy Service?